cost-analyzer/Chart.yaml

```diff
@@ -3,4 +3,4 @@ appVersion: "1.61.2"
 description: A Helm chart that sets up Kubecost, Prometheus, and Grafana to monitor
   cloud costs.
 name: cost-analyzer
-version: 1.61.2
+version: 1.61.3
```

Comment:
We don't want to bump here-- we batch things together for releases and bump all at once.